internal/k8s/watcher.go

```diff
@@ -37,7 +39,19 @@ func WatchCronJobs(g *workgroup.Group, client *kubernetes.Clientset, log logrus.
 }
 
 func watch(g *workgroup.Group, c cache.Getter, log logrus.FieldLogger, resource string, objType runtime.Object, rs ...cache.ResourceEventHandler) {
-	lw := cache.NewListWatchFromClient(c, resource, v1.NamespaceAll, fields.Everything())
+	//Check if the env var RESTRICTED_NAMESPACE is empty or equal to keel
+	// If equal to keel or empty, the scan will be over all the cluster
+	// If RESTRICTED_NAMESPACE is different than keel or empty, keel will scan in the defined namespace
+	namespace_scan := "keel"
```

Comment:
could you rename `namespace_scan` to `namespaceScan` ? (it's go convention)

did this approach work for you?

Reply:
Yes, for sure, I forgot that the convention is camelCase with go . This is done !